src/main/java/net/snowflake/client/core/client_config/SFClientConfig.java

```diff
@@ -0,0 +1,81 @@
+package net.snowflake.client.core.client_config;
```

Comment:
just double check underscore is ok in java naming convention.

Reply:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html it seems like `_` is allowed in the package name. Do you recommend `clientconfig` instead?